src/style.css

```diff
@@ -1 +1,16 @@
 /* styles go here */
+/* body {
+  margin: 0;
+} */
+
+/* label {
+  margin-top: 10px;
+} */
+
+fieldset {
```

Comment:
don't style tags

Reply:
didn't fix it. use classnames instead of tag selector

Reply:
And here you can use class name selector:
```
.fieldset {
  margin-bottom: 20px;
}
```

Reply:
this comment is still relevant
if you have questions add them to chat